external/CMakeLists.txt

```
@@ -50,14 +50,18 @@ add_library(ngtcp2::crypto
     ngtcp2_crypto)  # ngtcp2 crypto (and implicitly base ngtcp2)
 
 # oxen-logging
-if(BUILD_STATIC_DEPS)
-    set(OXEN_LOGGING_FORCE_SUBMODULES ON CACHE INTERNAL "")
+if (NOT TARGET oxen-logging)
```

`oxen-logging` is a semi-internal target for oxen-logging; the public (ALIAS) target is `oxen::logging`

```suggestion
if (NOT TARGET oxen::logging)
```